newIDE/app/src/MainFrame/Preferences/PreferencesContext.js

```diff
@@ -147,8 +148,14 @@ export const allAlertMessages: Array<{
     label: <Trans>Lifecycle functions only included when extension used</Trans>,
   },
   {
-    key: 'p2p-broker-recommendation',
-    label: <Trans>Peer to peer broker server recommendation</Trans>,
+    key: 'p2p-is-networking',
+    label: (
+      <Trans>Peer to peer IP address leak warning/THNK recommendation</Trans>
```

Comment:
Is it the text you want to display to users?

Reply:
This is the toggle in the settings to disable/reenable the hint

Reply:
Oh yeah, I forgot about that!